doc/tutorial/Tutorial.tex

```diff
@@ -859,15 +859,15 @@ \subsection{Paradoxes of classical predicate calculus}
 Check drinker.
 \end{coq_example}
 
-Remark how the three theorems are completely generic in the most general 
+Note how the three theorems are completely generic in the most general 
 fashion;
 the domain \verb:D: is discharged in all of them, \verb:R: is discharged in
 \verb:refl_if: only, \verb:P: is discharged only in \verb:weird: and
 \verb:drinker:, along with the hypothesis that \verb:D: is inhabited. 
 Finally, the excluded middle hypothesis is discharged only in 
 \verb:drinker:.
 
-Note that the name \verb:d: has vanished as well from
+Note, too, that the name \verb:d: has vanished from
```

Comment:
This changes the meaning of the sentence.

_reads context_

Ok, so the new meaning was the intended one. That's a rather important fix. Thanks for noticing that.